lib/extensions/lettering.py

```diff
@@ -173,6 +175,20 @@ def on_font_changed(self, event=None):
         font = self.fonts.get(self.font_chooser.GetValue(), self.default_font)
         self.settings.font = font.id
         self.scale_spinner.SetRange(int(font.min_scale * 100), int(font.max_scale * 100))
+
+        # Update the back_and_forth checkbox according to the chosen font
+        available_variants = font.available_variants()
+        if font.reversible:
+            self.back_and_forth_checkbox.Enable()
+            if FontVariant.reversed_variant(font.default_variant) in available_variants:
```

Comment:
I like how you used `reversed_variant` here, but I think we might need something even more generalized.  Eventually the Lettering extension will allow vertical text as well, and then we'll need to set the checkbox based on whether the top-to-bottom and bottom-to-top variants are both in the font.

This is starting to get really complicated.  Maybe we should just set this checkbox to unchecked by default and leave it at that?

Reply:
Maybe it's the default which we shouldn't be using? If we go up and down we still would want to know about the reversed version?

But I see why you think this is going to be complicated. And all this checking and unchecking could become confusing to users who do not know the reason why it's doing it. @LaureFR38 What would you think if we just left it unchecked all the time?

Reply:
I think that if it stays unchecked all the time people won't necessarily think of checking it. Also he will not understand why by checking it in some cases the reverse works well (because the file exists) and in other cases not. That's why I thought that checked or unchecked depending on the font let the user understand that the font was not always intended for the opposite direction.